assets/css/theme.css

```diff
@@ -250,12 +250,16 @@ Styles for the "current implementations" column
 
 /* UA filtering menu */
 [data-col|=impl] details {
+  display: none;
   position: relative;
-  display: block;
   padding: 0 0.1em;
   text-align: left;
 }
 
+[data-col|=impl] details.active {
```

Comment:
Why not just use `[data-col=impl]` here?

Reply:
There are two types of implementation columns: "Current implementations" and "Implementation intents". Internally, the `data-col` value for the former column is `impl` and the value for the latter is `impl-intents`. Styles need to apply to both columns, and that's precisely what the [`att|=val`](https://www.w3.org/TR/selectors-3/#attribute-representation) selector allows here!